projectroles/tests/test_templatetags.py

```diff
@@ -618,3 +618,75 @@ def test_get_sidebar_app_legend(self):
         self.assertEqual(
             tags.get_sidebar_app_legend('Update Project'), 'Update<br />Project'
         )
+
+    def test_get_sidebar_links_homepage(self):
```

Comment:
"Homepage" isn't a standard term in the project. "Home view" can be used.

We should also have tests for other resolvers, such as the project detail view, role views (to ensure the active checks work) and some app view (example project app is usually used for this).